### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by zanderz**
This package didn't work on Windows before. What do we do, just use our fork from now on? submit a request to the author?
@maxtaco @gabriel

### CODE REVIEW

**Comment by maxtaco** [comment]
Seem as if we've already forked it, so let's just apply these patches to our fork?

File: log_windows.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	}
+	return syscall.EINVAL
+}
+
+// use is a no-op, but the compiler cannot see that it is.
+// Calling use(p) ensures that p is kept live until that point.
+func use(p interface{}) {}
+
+func doFmtVerbLevelColor(layout string, level Level, output io.Writer) {
+	// no-op for now. We need the file descriptor to do colors.
```
Comment (maxtaco) [inline log_windows.go:122]: Does this mean that coloring isn't working currently for windows?

**Comment by zanderz** [comment]
I created this fork to at least do the code review internally. Any reason not to also submit to the package author?

**Comment by maxtaco** [comment]
oh right, sure, it's worth remerging in if we can..

**Pull request merged**